PROJECT

```diff
@@ -82,4 +82,28 @@ resources:
   kind: Environment
   path: github.com/redhat-appstudio/application-api/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
`DeploymentTargetClass` should be cluster-scoped, rather than namespace-scoped, it looks like.